### ISSUE TITLE
sparse_categorical_crossentropy of keras is not available

### ISSUE THREAD

**Issue by raytroop** [tensorflow/tensorflow#21735]
https://github.com/tensorflow/tensorflow/blob/05f8ea8e9522a3027d4f3f7a54d716bfafed427a/tensorflow/python/keras/metrics.py#L584-L591

I don't know why `sparse_categorical_crossentropy` is NOT decorated by `tf_export` like `binary_accuracy`, `sparse_top_k_categorical_accuracy` and so on.
As a result , we have to explicitly import `sparse_categorical_crossentropy`
 i.e. `from tensorflow.keras.metrics import sparse_categorical_crossentropy`

*additional information*:
Have I written custom code: No
OS Platform and Distribution:  Ubuntu 16.04.5 LTS
TensorFlow installed from: pypi
TensorFlow version: tensorflow-gpu 1.10.0
Bazel version: 0.15.0
CUDA/cuDNN version: CUDA 9.0, cuDNN v7.2.1 
GPU model and memory: nvidia GTX 1080TI, 11GB GDDR5
 Exact command to reproduce: `from tensorflow import keras; metric_func =keras.metrics.sparse_categorical_crossentropy`
Mobile device: N/A
Thanks

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by raytroop** [comment]
@tensorflowbutler I have complemented some information. Thanks

**Comment by facaiy** [comment]
It seems a mistake. Would you like to create a Pull Request to fix it?

**Comment by raytroop** [comment]
@facaiy  Thanks I will pull request soon

**Comment by tensorflowbutler** [comment]
Nagging Assignee @tatianashp: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
